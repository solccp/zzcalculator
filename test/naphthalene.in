   10
12
 C     0.000000     0.000000     1.400069
 C     1.212495     0.000000     0.700034
 C     1.212495     0.000000    -0.700035
 C     0.000000     0.000000    -1.400069
 C    -1.212495     0.000000    -0.700035
 C    -1.212495     0.000000     0.700035
 C     2.424991     0.000000    -1.400069
 C     3.637486     0.000000    -0.700035
 C     2.424991     0.000000     1.400069
 C     3.637486     0.000000     0.700034
