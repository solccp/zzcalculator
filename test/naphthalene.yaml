---
structure_id: 1
number_of_atoms: 10
double_bonds:
- [1, 6]
- [5, 4]
- [10, 9]
- [8, 7]
- [3, 2]
---
structure_id: 2
number_of_atoms: 10
double_bonds:
- [1, 6]
- [5, 4]
- [10, 8]
- [7, 3]
- [2, 9]
---
structure_id: 3
number_of_atoms: 10
double_bonds:
- [1, 6]
- [5, 4]
rings:
- [10, 8, 7, 3, 2, 9]
---
structure_id: 4
number_of_atoms: 10
double_bonds:
- [1, 2]
- [6, 5]
- [4, 3]
- [7, 8]
- [10, 9]
---
structure_id: 5
number_of_atoms: 10
double_bonds:
- [7, 8]
- [10, 9]
rings:
- [1, 2, 3, 4, 5, 6]
