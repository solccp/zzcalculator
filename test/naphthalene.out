 3 + 2 x
 total: 5
